lib/active_stash/railtie.rb

```diff
@@ -1,3 +1,5 @@
+require 'rails'
```

Comment:
There's no dependency on `rails` in the gemspec; this will probably assplode for anyone using ActiveRecord without Rails.

Reply:
`railtie.rb` is not required unless running as part of a Rails app. See `active_stash.rb`

```
# snip
require "active_stash/railtie" if defined?(Rails::Railtie)
# snip
```